src/main/java/software/amazon/cloudformation/proxy/HandlerRequest.java

```diff
@@ -34,5 +34,6 @@
     private RequestData<ResourceT> requestData;
```

Comment:
nit: should this be in request data instead? seems relatedto the actual request and not cloudformation metadata related info

Reply:
Good point, but TBH I don't really understand the distinction here anyways.  Like, why are providerCredentials, providerLogGroupName, and logicalResourceId part of requestData while callbackContext, accountId, bearerToken, and action are not?